_redirects

```diff
@@ -382,7 +382,6 @@ layout: null
 /docs/cockroachcloud/migrate-from-free-to-dedicated /docs/cockroachcloud/migrate-from-serverless-to-dedicated.html 301
 /docs/cockroachcloud/free-faqs /docs/cockroachcloud/serverless-faqs.html 301
 /docs/v22.2/use-cloud-storage-for-bulk-operations.html /docs/v22.2/use-cloud-storage.html 301
-/docs/v23.1/use-cloud-storage-for-bulk-operations.html /docs/v23.1/use-cloud-storage.html 301
```

Comment:
Any reason you're moving the redirect here? I think it's still necessary to include.

Reply:
added back above